services/quests/errors/definitions/427.tsx

```diff
@@ -0,0 +1,136 @@
+export const NUMBER = 427;
+export const NAME = `Detected an invalid transition from this combat to another combat`;
```

Comment:
Can be streamlined to `Invalid transition from this...`

Reply:
Done.